release/release.js

```diff
@@ -255,6 +256,24 @@ async function releaseIOSSpecs() {
     await runCmds(path.join(config.tmpDir, repo), cmds)
 }
 
+//
+// Release function for iOS-Spm repo
+//
+async function releaseIOSSpm() {
+    const repo = REPO.iosspecs
+    const cmds = {
+        msg: `PROCESSING ${repo}`,
+        cmds: [
+            cloneOrClean(config.org, repo, config.tmpDir),
+            `git checkout ${config.masterBranch}`,
+	    `build_xcframeworks.sh -r {config.org} -b ${config.masterBranch}`,
+            commitAndPushMaster(),
+	    tagMaster(true) // SPM needs versions of the form X.Y.Z (where X, Y, Z are integers)
```

Comment:
The true argument means that no prefix should be used for the version tag. 
When adding a package with SPM by version (i.e. tag) in Xcode, only X.Y.Z (with integers) are supported.

Reply:
At release time, we will run the script that build xcframeworks pointing to the master branch of the main repo, then check them in and finally tag the repo.